packages/run-protocol/src/contractSupport.js

```diff
@@ -1,4 +1,5 @@
 // @ts-check
+import { q } from '@agoric/assert';
```

Comment:
I don't think I should approve code with this idiom. Your point that the deprecation isn't documented is well made, but I still think it's the position of several in the team, and as a reviewer, I'm obliged to represent them.

Reply:
Deferring resolution, moved to chat